api/models/userDao.js

```diff
@@ -0,0 +1,17 @@
+const appDataSource = require("./dataSource");
+
+const createUser = async (name, email, password, point) => {
+  await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      password,
+      point )
+      VALUES(?,?,?,?)`,
+    [name, email, password, point]
+  );
```

Comment:
```suggestion
  return appDataSource.query(
    `INSERT INTO users(
      name,
      email,
      password,
      point )
      VALUES(?,?,?,?)`,
    [name, email, password, point]
  );
```
함수의 실행문 내부에서 동기적으로 수행할 추가적인 로직이 없기 떄문에 await가 없어도 됩니다. 또한 함수 내부에 return이 없어도 되지만 model layer에서는 mysql 에서 만들어주는 okpacket 객체를 활용하는 경우(ex. insertId)도 있기 때문에 return을 넣어서 작성할 수 있도록 해주세요~